### PR TITLE
apycula: init at 0.0.1a9

### PR DESCRIPTION
**Pull request by newAM**
###### Motivation for this change

Add [apycula](https://github.com/YosysHQ/apicula), open source tools for working with Gowin FPGAs.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by risicle** [comment]
@ofborg build apycula

File: pkgs/development/python-modules/apycula/default.nix [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+, setuptools-scm
+, numpy
+, pandas
+, pillow
+, crcmod
+, openpyxl
+}:
+
+buildPythonPackage rec {
+  pname = "Apycula";
```
Comment (fabaff) [inline pkgs/development/python-modules/apycula/default.nix:14]:
```suggestion
  pname = "apycula";
```

see https://nixos.org/manual/nixpkgs/stable/#contributing

Comment (newAM) [reply pkgs/development/python-modules/apycula/default.nix]: Fixed.  Thank you for the review!

**Comment by figsoda** [comment]
@ofborg build python3Packages.apycula

**Comment by newAM** [comment]
Looks like the `x86_64-darwin` failure is related to: https://github.com/NixOS/nixpkgs/pull/137870

**Review by figsoda — APPROVED** [review]

**Comment by figsoda** [comment]
Result of `nixpkgs-review pr 138450` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python38Packages.apycula</li>
    <li>python39Packages.apycula</li>
  </ul>
</details>

**Pull request merged**